### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: demisto/etc/issues/29346

## Description
replaced `-p` flag with `-i`

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/release-notes.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Release notes files help to keep track of the changes made for a specific conten
 To generate a release notes markdown file, first commit the changes to your branch and then run the following command provided by the `demisto-sdk`:
 
 ```bash
-demisto-sdk update-release-notes -p [Changed Pack Name] -u [major|minor|revision]
+demisto-sdk update-release-notes -i [Changed Pack Name] -u [major|minor|revision]
```
Comment (Shellyber) [inline docs/integrations/release-notes.md:11]: Maybe better to change it to [Changed pack path]

File: docs/integrations/release-notes.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ changes *after* running the command, you will need to remove the `-u` argument.
 file for you to fill out.
 
 ```bash
-demisto-sdk update-release-notes -p [Changed Pack Name]
+demisto-sdk update-release-notes -i [Changed Pack Name]
```
Comment (Shellyber) [inline docs/integrations/release-notes.md:23]: Same here

**Comment by Itay4** [comment]
@Bargenish i think you can ignore the Circle CI build for now, its not required
@glicht can you confirm ?

**Comment by glicht** [comment]
@Bargenish you can ignore the circleci fail. It is not setup yet.

**Review by glicht — APPROVED** [review]

**Pull request merged**